ovirt/data_source_ovirt_vms.go

```diff
@@ -199,7 +205,7 @@ func dataSourceOvirtVMsRead(d *schema.ResourceData, meta interface{}) error {
 		filteredVMs = vms.Slice()[:]
 	}
 
-	if len(filteredVMs) == 0 {
+	if len(filteredVMs) == 0 && allowEmptyEesult == false {
```

Comment:
consider using ! operator here ?

Reply:
Of course, I tried this at the beginning:
`if len(filteredVMs) == 0 && !allowEmptyEesult {`

But GO compilator return error in this case:
```bash
# GOOS=linux GOARCH=amd64 go build
# github.com/ovirt/terraform-provider-ovirt/ovirt
ovirt/data_source_ovirt_vms.go:208:30: invalid operation: ! interface {}
```
Therefore I forced to use syntax: allowEmptyEesult == false